rs-matter-macros/src/idl/cluster.rs

```diff
@@ -446,8 +445,17 @@ pub fn cluster(cluster: &Cluster, context: &IdlGenerateContext) -> TokenStream {
 
     let cluster_id = Literal::u32_unsuffixed(cluster.code as u32);
     let cluster_revision = Literal::u16_unsuffixed(cluster.revision as u16);
+    let import_globals = if !globals.enums.is_empty()
```

Comment:
FYI: This code has a trivial mistake: it is using `&&` instead of `||`, but there are likely other reasons why it does not kick in.